### PR TITLE
Add lavfilters702 and wsh57

### PR DESCRIPTION
**Pull request by Amb0s**
## Type of change
- [x] New dependency
- [ ] Manifest fix
- [ ] Other

# Was this tested using a [local repository](https://maintainers.usebottles.com/Testing)?
- [x] Yes
- [ ] No

Based on the Winetricks implementations of [LAV Filters 0.70.2](https://github.com/Winetricks/winetricks/blob/9899670f459425eade737a249b775f966ac63647/src/winetricks#L11211) and [MS Windows Script Host 5.7](https://github.com/Winetricks/winetricks/blob/9899670f459425eade737a249b775f966ac63647/src/winetricks#L13752).

Both dependencies were tested with caffe-7.7 in a 64-bit bottle.

### CODE REVIEW

File: Essentials/wsh57.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+Name: wsh57
+Description: MS Windows Script Host 5.7
+Provider: Microsoft
+License:
+License_url:
```
Comment (orowith2os) [inline Essentials/wsh57.yml:5]: Missing a license.

**Review by orowith2os — COMMENTED** [review]
LGTM at first glance (other than the missing license), will let Mirko or another maintainer merge it.

**Comment by Amb0s** [comment]
I didn't find any link to the licence on the [download page](https://www.microsoft.com/en-us/download/details.aspx?id=8247).

**Comment by mirkobrombin** [comment]
Set the license to "Microsoft" and the url to the download link

**Comment by Amb0s** [comment]
Done.

**Comment by mirkobrombin** [comment]
Thanks

**Pull request merged**